supplier/tests/test_supplier_page.py

```diff
@@ -13,6 +35,36 @@ def non_supplier_mock():
     return mock
 
 
+@pytest.fixture
+def request_searched_product_mock():
+    mock = Mock()
+    mock.GET = {}
+    return mock
+
+
+@pytest.fixture
+def render_mock(monkeypatch):
+    mock = Mock()
+    monkeypatch.setattr(views, views.render.__name__, mock)
+    return mock
+
+
+@pytest.fixture
+def messages_mock(monkeypatch):
+    mock = Mock()
+    monkeypatch.setattr(views, "messages", mock)
+    return mock
+
+
+@pytest.fixture
+def all_products_mock(saved_supplier0,
+                      saved_supplier_product0,
+                      saved_supplier_product1):
+    mock = Mock()
+    mock.return_value = SupplierProduct.objects.filter(user_name=saved_supplier0)
+    return mock
```

Comment:
I don't see it being used. If it is to be used, so I think it can be deleted.
In any case, typically, mocks are there when you don't want or can't make the real calls. In this case, the mock's return value is the output of an actual call. I don't think such use case requires a mock.

Reply:
deleted the test it was used, will delete this block too👍